source/server/http/admin.cc

```diff
@@ -1009,7 +1009,12 @@ Http::Code AdminImpl::handlerAdminHome(absl::string_view, Http::HeaderMap& respo
 
   // Prefix order is used during searching, but for printing do them in alpha order.
   for (const UrlHandler* handler : sortedHandlers()) {
-    const std::string& url = handler->prefix_;
+    absl::string_view path = handler->prefix_;
+    // Removing the leading slash from the link, so that the admin page can be
```

Comment:
Removing -> remove.  Also as someone not terribly familiar with this code this comment doesn't quite explain what's going on - could we spell it out a bit more clearly?

Reply:
done; ptal